[2024-04-14T19:24:47.272+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-14T19:24:47.280+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: upload_csv_to_s3.generate_and_upload_csv manual__2024-04-14T19:24:44.748380+00:00 [queued]>
[2024-04-14T19:24:47.285+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: upload_csv_to_s3.generate_and_upload_csv manual__2024-04-14T19:24:44.748380+00:00 [queued]>
[2024-04-14T19:24:47.285+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-04-14T19:24:47.291+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): generate_and_upload_csv> on 2024-04-14 19:24:44.748380+00:00
[2024-04-14T19:24:47.297+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=2874) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-04-14T19:24:47.300+0000] {standard_task_runner.py:63} INFO - Started process 2876 to run task
[2024-04-14T19:24:47.300+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'upload_csv_to_s3', 'generate_and_upload_csv', 'manual__2024-04-14T19:24:44.748380+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/test_dag.py', '--cfg-path', '/tmp/tmpowtvb10w']
[2024-04-14T19:24:47.301+0000] {standard_task_runner.py:91} INFO - Job 28: Subtask generate_and_upload_csv
[2024-04-14T19:24:47.325+0000] {task_command.py:426} INFO - Running <TaskInstance: upload_csv_to_s3.generate_and_upload_csv manual__2024-04-14T19:24:44.748380+00:00 [running]> on host 514aa00dce91
[2024-04-14T19:24:47.362+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='upload_csv_to_s3' AIRFLOW_CTX_TASK_ID='generate_and_upload_csv' AIRFLOW_CTX_EXECUTION_DATE='2024-04-14T19:24:44.748380+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-14T19:24:44.748380+00:00'
[2024-04-14T19:24:47.363+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-14T19:24:48.070+0000] {logging_mixin.py:188} INFO - []
[2024-04-14T19:24:48.071+0000] {logging_mixin.py:188} INFO - Processing section: 16
[2024-04-14T19:24:50.266+0000] {base.py:84} INFO - Using connection ID 'aws_default' for task execution.
[2024-04-14T19:24:50.268+0000] {connection_wrapper.py:384} INFO - AWS Connection (conn_id='aws_default', conn_type='aws') credentials retrieved from login and password.
[2024-04-14T19:24:50.833+0000] {logging_mixin.py:188} INFO - File uploaded successfully to verdemar-dataset as products_20240414.csv
[2024-04-14T19:24:50.835+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-04-14T19:24:50.836+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-14T19:24:50.855+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=upload_csv_to_s3, task_id=generate_and_upload_csv, execution_date=20240414T192444, start_date=20240414T192447, end_date=20240414T192450
[2024-04-14T19:24:50.933+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-14T19:24:50.971+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-14T19:24:50.975+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
